src/psyclone/psyir/nodes/reference.py

```diff
@@ -64,6 +64,14 @@ def __init__(self, symbol, parent=None):
         super(Reference, self).__init__(parent=parent)
         self.symbol = symbol
 
+    def is_array(self):
```

Comment:
For consistency with the Signature.is_structure method, I suggest making this a property (and also in ArrayMixin).

Reply:
Done.